lib/view_component/base.rb

```diff
@@ -411,6 +411,8 @@ def iteration_argument_present?
       private
 
       def initialize_parameter_names
```

Comment:
Might we rename this to `parameter_names` as this is not just for the initializer now?

Reply:
I suppose there's a case for renaming it, but `parameter_names` is unqualified, so it's not as descriptive. The return value of this method returns the valid parameters for the method - even though something that includes Active Model takes a params hash instead of explicit keyword arguments, these are the only ones we expect to be valid. So perhaps `valid_initialize_parameter_names` is the right name for this now?

Reply:
Let's just leave it as is 😄